frontend/src/components/Content.js

```diff
@@ -0,0 +1,23 @@
+import { Container, Typography } from "@mui/material";
+import React, { useContext } from "react";
+import { UserInfoContext } from "../stores/UserInfoStore";
+
+const Content = () => {
+  const [userInfo, userInfoDispatch] = useContext(UserInfoContext);
+
+  return (
+    <Container sx={{ clear: "right", ml: "0" }}>
+      <Typography variant="h4">
+        {userInfo.loggedIn ? (
+          <>
+            Hello {userInfo.role.toLowerCase()}, {userInfo.email}{" "}
+          </>
+        ) : (
+          "Hello visitor."
+        )}
+      </Typography>
+    </Container>
+  );
+};
```

Comment:
Where are the child components going to be displayed ?
The content component should be a container (the content area) to the currently displayed work space.
See UserInfoStore for child component example.
Call me on discord for questions.